### PR TITLE
WPScan January Public CVEs

### PR DESCRIPTION
**Pull request by ethicalhack3r**
First time doing this, please let me know if there's something we have done incorrectly, or something that we can improve.

This first batch was done manually, but we will aim to automate the next batch.

### CODE REVIEW

**Comment by cve-team** [comment]
ethicalhack3r is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2021/24xxx/CVE-2021-24123.json [hunk 1]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24123.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24123.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24124.json [hunk 2]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24124.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24124.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24125.json [hunk 3]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24125.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24125.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24126.json [hunk 4]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24126.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24126.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24127.json [hunk 5]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24127.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24127.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24128.json [hunk 6]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24128.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24128.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24129.json [hunk 7]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24129.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24129.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24130.json [hunk 8]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24130.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24130.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24131.json [hunk 9]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24131.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24131.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24132.json [hunk 10]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24132.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24132.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24133.json [hunk 11]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24133.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24133.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24134.json [hunk 12]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24134.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24134.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24135.json [hunk 13]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24135.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24135.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24136.json [hunk 14]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24136.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24136.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24137.json [hunk 15]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24137.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24137.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24138.json [hunk 16]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24138.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24138.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24139.json [hunk 17]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24139.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24139.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24140.json [hunk 18]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24140.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24140.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24141.json [hunk 19]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24141.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24141.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24142.json [hunk 20]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24142.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24142.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24143.json [hunk 21]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24143.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24143.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24144.json [hunk 22]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24144.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24144.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24145.json [hunk 23]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24145.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24145.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24146.json [hunk 24]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24146.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24146.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24147.json [hunk 25]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24147.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24147.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24148.json [hunk 26]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24148.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24148.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24149.json [hunk 27]
```diff
@@ -1,18 +1,74 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24149.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24149.json:1]: 'version_value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by ethicalhack3r** [comment]
I'll get these fixed next week

**Comment by ethicalhack3r** [comment]
Adding `Unknown` as `vendor_name` value as each plugin has a different vendor, which we don't record

File: 2021/24xxx/CVE-2021-24123.json [hunk 28]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24123.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24124.json [hunk 29]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24124.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24125.json [hunk 30]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24125.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24126.json [hunk 31]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24126.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24127.json [hunk 32]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24127.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24128.json [hunk 33]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24128.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24129.json [hunk 34]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24129.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24130.json [hunk 35]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24130.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24131.json [hunk 36]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24131.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24132.json [hunk 37]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24132.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24133.json [hunk 38]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24133.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24134.json [hunk 39]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24134.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24135.json [hunk 40]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24135.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24136.json [hunk 41]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24136.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24137.json [hunk 42]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24137.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24138.json [hunk 43]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24138.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24139.json [hunk 44]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24139.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24140.json [hunk 45]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24140.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24141.json [hunk 46]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24141.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24142.json [hunk 47]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24142.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24143.json [hunk 48]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24143.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24144.json [hunk 49]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24144.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24145.json [hunk 50]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24145.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24146.json [hunk 51]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24146.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24147.json [hunk 52]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24147.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24148.json [hunk 53]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24148.json:1]: 'version_value' is a required property

File: 2021/24xxx/CVE-2021-24149.json [hunk 54]
```diff
@@ -1,18 +1,75 @@
...
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24149.json:1]: 'version_value' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by ethicalhack3r** [comment]
I think this should be ready to merge now?

**Comment by lsekela** [comment]
The prose descriptions supplied are missing any product/vendor information. A CVE description (8.2.1) MUST provide enough information for a reader to have a reasonable understanding of what products are affected. You can find requirements and an example of prose descriptions at https://cve.mitre.org/cve/list_rules_and_guidance/cve_assignment_information_format.html

**Comment by cristina479** [comment]
Please fix the inconsistencies between the versions in the description and the version information in the `version` field for the CVEs. As an example, the description of CVE-2021-24125 indicates that versions less than 1.6.4 (not including 1.6.4) are affected but the version field indicates that version <= 1.6.4 are affected. Thank you.

**Comment by ethicalhack3r** [comment]
Thank you Christina, I'll review and get that fixed tomorrow.

Please excuse us for the mistakes, this is our first time submitting but plan to automate this process in the future, which should hopefully reduce the mistakes.

**Comment by cristina479** [comment]
> Thank you Christina, I'll review and get that fixed tomorrow.
> 
> Please excuse us for the mistakes, this is our first time submitting but plan to automate this process in the future, which should hopefully reduce the mistakes.

No problem. Glad to help.

**Comment by ethicalhack3r** [comment]
I think this should be OK now?

**Pull request merged**